ranger/tmpl/error.tmpl

```diff
@@ -36,7 +35,9 @@
     padding: .25rem .5rem;
   }
   #logo {
-    margin: 0 0 2rem 100px; 
+    margin: 2rem 120px; 
+    width: 150px;
+    height: auto;
   }
   #sash {
     background: #1F6DF4;
```

Comment:
Since you're already in here, want to swap the old blue for the new `#2850BE`?

Reply:
good catch!